docs/reference/api/storage/HeapBlockDevice.md

```diff
@@ -1,2 +1,20 @@
 ## HeapBlockDevice
 
+The HeapBlockDevice class provides a way to simulate block devices for software development or testing. The created blocks are nonvolatile; they do not persist across power cycles.
+
+HeapBlockDevices have some notable configurable parameters:
+
+  - _size_ - Size of the block device in bytes.
```

Comment:
How are these configured? Is this just the constructor? Why are they notable?

Reply:
Correct, they're constructor parameters. Wanted to list them out up front, but they are listed in attached class reference links as well. The phrasing is probably a bit off, they're not so much notable parameters as much as a list of parameters used to configure the HeapBlockDevice. I can change the wording or pull that list out.

Reply:
We have a planned section for configuration options for storage APIs (https://jira.arm.com/browse/IOTMORF-1273). However, this planned page should be generated JSON. Would the generated JSON include this content?